module/VuFind/src/VuFind/Auth/LoginToken.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+/**
+ * Persistent login token manager
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  VuFind\Auth
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+declare(strict_types=1);
+
+namespace VuFind\Auth;
+
+use Laminas\Session\SessionManager;
+use VuFind\Exception\Auth as AuthException;
+use VuFind\Exception\LoginToken as LoginTokenException;
+
+/**
+ * Class LoginToken
+ *
+ * @category VuFind
+ * @package  VuFind\Auth
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class LoginToken implements \VuFind\I18n\Translator\TranslatorAwareInterface
```

Comment:
Maybe this class should be renamed to LoginTokenManager, since that's what it is; leaving off the "Manager" potentially makes it sound like a class that represents a single login token.

Reply:
Sure, LoginTokenManager is more descriptive. Renamed